src/main/java/su/terrafirmagreg/core/compat/kjs/events/TFGAE2PowerConsumption.java

```diff
@@ -0,0 +1,28 @@
+package su.terrafirmagreg.core.compat.kjs.events;
+
+import net.minecraft.resources.ResourceLocation;
+
+import dev.latvian.mods.kubejs.event.EventJS;
+import it.unimi.dsi.fastutil.objects.Object2ObjectOpenHashMap;
+
+import su.terrafirmagreg.core.TFGCore;
+
+public class TFGAE2PowerConsumption extends EventJS {
+    public static final Object2ObjectOpenHashMap<ResourceLocation, Double> powerConsumption = new Object2ObjectOpenHashMap<>();
+
+    public void add(ResourceLocation level, double power) {
```

Comment:
wouldn't the api be better if it just accepted a string, and then java turned it into a resource location?